frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
```

Comment:
The API class is not storing any state. Thus for ergonomics of usage, it could either have static methods or then it could only export certain functions. So the invocation could then be like for example

```
import { BackendApi } from './apis'; // Static methods on class
import * as BackendApi from './apis'; // Plain functions

const courses = await BackendApi.getCourses();
```